### PR TITLE
Simplify setup wizard: remove angular, combine all data on one page, etc

### PR DESCRIPTION
**Pull request by hws47a**
### Description (*)

Simplify setup wizard code:
- remove angular
- combine landing+license on the same page
- remove other unused files

This could be one of the steps to remove laminas MVC and other dependencies used only for setup scripts and which are not necessary after removing the main Setup Wizard logic

### Manual testing scenarios (*)

Open setup wizard `/setup/` and check that both landing and license pages are the same as before

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#32550: Simplify setup wizard: remove angular, combine all data on one page, etc

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@hws47a looks really good! Less code, fewer dependencies, more simple solutino is definitely a good thing! Let me know when I can start reviewing it.

@magento run Functional Tests B2B, Unit Tests

**Comment by hws47a** [comment]
@ihor-sviziev you can start, it doesn’t seem that any checks are failing because of my changes.

**Comment by sivaschenko** [comment]
@magento run Unit Tests

File: setup/src/Magento/Setup/Controller/Index.php [hunk 1]
```diff
@@ -8,19 +8,42 @@
...
+     * @var ProductMetadata
+     */
+    private $productMetadata;
+
+    /**
+     * @var License
+     */
+    private $license;
+
+    public function __construct(
```
Comment (sivaschenko) [inline setup/src/Magento/Setup/Controller/Index.php:37]: Please add the method phpdoc

File: setup/src/Magento/Setup/Controller/Index.php [hunk 2]
```diff
@@ -8,19 +8,42 @@
...
+    }
+
+    /**
+     * Setup index action.
      *
-     * @return ViewModel|\Laminas\Http\Response
+     * @return ViewModel
      */
-    public function indexAction()
+    public function indexAction(): ViewModel
```
Comment (sivaschenko) [inline setup/src/Magento/Setup/Controller/Index.php:50]: I'd also add `declare(strict_types=1);` to this file

File: setup/pub/scripts/main.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+function showSection(section) {
```
Comment (sivaschenko) [inline setup/pub/scripts/main.js:6]: Please add the copyrights section

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the pull request @hws47a ! Please see my review comments.

**Comment by hws47a** [comment]
@sivaschenko all three fixed

**Comment by hws47a** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Integration Tests

**Comment by ihor-sviziev** [comment]
@magento run Integration Tests

**Comment by hws47a** [comment]
@ihor-sviziev all passed 🙌

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8932](https://jira.corp.magento.com/browse/ENGCOM-8932) has been created to process this Pull Request

**Comment by hws47a** [comment]
@magento run all tests

**Review by ihor-sviziev — APPROVED** [review]

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests CE, Integration Tests

**Comment by ihor-sviziev** [comment]
@magento run Integration Tests

**Comment by sivaschenko** [comment]
@magento create issue

**Pull request merged**